util/lsp/src/hover.rs

```diff
@@ -134,6 +147,20 @@ impl ToHoverContent for AnnoInfo {
     }
 }
 
+impl ToHoverContent for LocalMatchInfo {
+    fn to_hover_content(self) -> HoverContents {
+        let header = MarkedString::String("Local match".to_owned());
```

Comment:
We probably want to display the type here.

Reply:
Fixed in [8152d23](https://github.com/polarity-lang/polarity/pull/205/commits/8152d236610b8ab6800a04176b5874b0a5b41b7f)